pegasus/migrations/80_add_purpose_and_region_columns_to_hoc_survey_prizes.rb

```diff
@@ -0,0 +1,18 @@
+Sequel.migration do
+  up do
+    alter_table(:hoc_survey_prizes) do
+      # can't be null for unique constraint to work. Use '' for unclaimed and legacy-claimed prizes
+      add_column :purpose, String, size: 255, null: false, default: ''
+      add_index [:claimant, :purpose], unique: true
+      drop_index :claimant
+    end
+  end
+
+  down do
+    alter_table(:hoc_survey_prizes) do
+      drop_column :purpose
```

Comment:
I tested the migrate and rollback locally several times. It does work. Maybe the order doesn't matter?